ui/src/main/webapp/app/graph/graph-component.js

```diff
@@ -630,9 +654,24 @@ function GraphController($q, graph, config, error, loading, query, operationOpti
      * Removes every selected element in the graph.
      */
     vm.removeSelected = function() {
-        cytoscapeGraph.filter(":selected").remove();
+        var removedElements = cytoscapeGraph.filter(":selected").remove();
+        for(var i = 0; i < removedElements.length; i++) {
```

Comment:
could you use array.concat() here? https://devdocs.io/javascript/global_objects/array/concat

Reply:
Unfortunately removedElements is not actually an array, so I can't use that.

Reply:
You could use the [toArray method](https://js.cytoscape.org/#eles.toArray)